doc-content/enterprise-only/openshift/rhdm71-authoring-ha-ref.adoc

```diff
@@ -0,0 +1,500 @@
+|=======================================================================
+
+
+
+== Objects
+
+The CLI supports various object types. A list of these object types as well as their abbreviations
+can be found in the https://docs.openshift.org/latest/cli_reference/basic_cli_operations.html#object-types[Openshift documentation].
+
+
+=== Services
+
+A service is an abstraction which defines a logical set of pods and a policy by which to access them. Refer to the
+https://cloud.google.com/container-engine/docs/services/[container-engine documentation] for more information.
+
+|=============
+|Service        |Port  |Name | Description
+
+.3+| `${APPLICATION_NAME}-rhdmcentr`
+|8080 | http
+.3+| All the Decision Central web server's ports.
+|8443 | https
+|8001 | git-ssh
+.1+| `${APPLICATION_NAME}-ping`
+|8888 | ping
+.1+| The JGroups ping port for clustering.
+.2+| `${APPLICATION_NAME}-kieserver`
+|8080 | http
+.2+| All the KIE server web server's ports.
+|8443 | https
+.2+| `${APPLICATION_NAME}-rhdmindex`
+|9200 | rest
+.2+| All the Decision Central Indexing Elasticsearch ports.
+|9300 | transport
+.1+| `${APPLICATION_NAME}-amq-tcp`
+|61616 | --
+.1+| The broker's OpenWire port.
+|=============
+
+
+
+=== Routes
+
+A route is a way to expose a service by giving it an externally-reachable hostname such as `www.example.com`. A defined route and the endpoints
+identified by its service can be consumed by a router to provide named connectivity from external clients to your applications. Each route consists
+of a route name, service selector, and (optionally) security configuration. Refer to the
+https://docs.openshift.com/enterprise/3.0/architecture/core_concepts/routes.html[Openshift documentation] for more information.
+
+|=============
+| Service    | Security | Hostname
+
+|`${APPLICATION_NAME}-rhdmcentr-http` | none | `${DECISION_CENTRAL_HOSTNAME_HTTP}`
+|`${APPLICATION_NAME}-rhdmcentr-https` | TLS passthrough | `${DECISION_CENTRAL_HOSTNAME_HTTPS}`
+|`${APPLICATION_NAME}-kieserver-http` | none | `${KIE_SERVER_HOSTNAME_HTTP}`
+|`${APPLICATION_NAME}-kieserver-https` | TLS passthrough | `${KIE_SERVER_HOSTNAME_HTTPS}`
+|`${APPLICATION_NAME}-rhdmindex-http` | none | `${ES_HOSTNAME_HTTP}`
+|=============
+
+
+
+
+=== Deployment Configurations
+
+A deployment in OpenShift is a replication controller based on a user defined template called a deployment configuration. Deployments are created manually or in response to triggered events.
+Refer to the https://docs.openshift.com/enterprise/3.0/dev_guide/deployments.html#creating-a-deployment-configuration[Openshift documentation] for more information.
```

Comment:
Search all "Refer to" and replace with "See", according to our team and IBM SG guidelines for consistency.

Reply:
Autogeneration again